modules/test-structure/save_test_data_test.go

```diff
@@ -218,3 +219,23 @@ func TestSaveAndLoadNamedInts(t *testing.T) {
 	assert.Equal(t, expectedData1, actualData1)
 	assert.Equal(t, expectedData2, actualData2)
 }
+
+func TestSaveAndLoadKubectlOptions(t *testing.T) {
+	t.Parallel()
+
+	tmpFolder, err := ioutil.TempDir("", "save-and-load-kubectl-options")
+	if err != nil {
+		t.Fatalf("Failed to create temp dir: %v", err)
+	}
+
+	expectedData := &k8s.KubectlOptions{
+		ContextName: "terratest-context",
+		ConfigPath:  "~/.kube/config",
+		Namespace:   "default",
+		Env:         map[string]string{},
```

Comment:
Can you also populate `Env` with a few entries to make sure the nested object serializes correctly?